src/android/com/adobe/phonegap/push/PushConstants.java

```diff
@@ -7,6 +7,10 @@
   public static final String FOREGROUND = "foreground";
   public static final String TITLE = "title";
   public static final String NOT_ID = "notId";
+  public static final String GROUP_ID = "groupId";
+  public static final String TO_DELETE_ID = "toDeleteId";
+  public static final String IS_DELETING = "deleting";
```

Comment:
```suggestion
  public static final String IS_DELETING = "isDeleting";
```

To be consistent with other constants.